DESCRIPTION

```diff
@@ -53,7 +53,8 @@ Imports:
     parallelMap (>= 1.3),
     survival,
     utils,
-    XML
+    XML,
+    mmpf
```

Comment:
Packages here should be in alphabetical order.

Reply:
ok will do